stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -1673,6 +1868,41 @@ pub fn simple_nakamoto_coordinator_10_extended_tenures_10_sortitions() -> TestPe
 
         debug!("Next burnchain block: {}", &consensus_hash);
 
+        let block_height = peer
```

Comment:
Seems this could be DRY'ed up with the code at line 1140.

Reply:
Good idea. Refactored in 0fdaf62fb.